rtc-video-quality/README.md

```diff
@@ -0,0 +1,185 @@
+# Measuring Video Codec Performance
+
```

Comment:
You may want to make a note here that you've modified the files from the original rtc-video-quality repository (and link to the repository as well).

You should also add a comment in the scripts that you've modified noting that they've been modified by you from the original version as well.

Reply:
Updated readme and the [modified files](https://github.com/googleinterns/av1-codec-comparison/pull/1/commits/61abaebaafcc2384a84540adfe201c020753d49d)